src/role_scout.js

```diff
@@ -9,6 +9,7 @@
 roles.scout = {};
 roles.scout.settings = {
   layoutString: 'M',
+  amount: [1],
```

Comment:
I'm not 100% sure if it is still needed, but making it explicit is surely not bad